tests/e2e/script-test-consensus.test.ts

```diff
@@ -0,0 +1,47 @@
+import { RPCHandler } from "../../dist";
+import { RequestPayload } from "../../dist/types/rpc-service";
```

Comment:
Import from `types` and not `dist`. Breakpoints are not working with this.

Reply:
@EresDev I'm confused now because the purpose of the scripts and subsequently these tests were to test the build, the final `/dist` output that is consumed by package installers. The unit tests all import from `/types`, between the two the only difference of what's being tested was the fact is was `/dist` imports.

Reply:
I thought unit tests are using anvil, but e2e are using production blockchain. Isn't it so? 

An e2e test should import how the end user will import.

Reply:
Ah okay I see the confusion. 

The write-ops all use anvil yeah but where possible, all other tests use production blockchain. If the config has `networkId: 31337` then it's using anvil, `100` would be real APIs.